tests/test_api/test_submit_job.py

```diff
@@ -263,6 +264,8 @@ def test_submit_good_autorift_granule_names(client, tables):
         'S2B_22WEB_20200612_0_L1C',
         'LC08_L1TP_009011_20200820_20200905_02_T1',
         'LO08_L1GT_043001_20201106_20201110_02_T2',
+        'LC09_L1GT_215109_20220125_20220125_02_T2',
+        'LC09_L1GT_215109_20220210_20220210_02_T2',
```

Comment:
should try sensor mode `LO09` for the second scene here

Reply:
```suggestion
        'LO09_L1GT_215109_20220210_20220210_02_T2',
```